site2/docs/client-libraries-cpp.md

```diff
@@ -394,22 +394,6 @@ Consumer consumer;
 client.subscribe("my-topic", "my-sub", conf, consumer);
 ```
 
-## Enable authentication in connection URLs
```

Comment:
Why is this whole section removed?

Reply:
Dave, thanks for asking. The reasons include:
* Lack of consistency: this section is not available in other clients' docs even though it applies.
* Duplicate: somehow duplicate with the content in the [Connection URL](https://pulsar.apache.org/docs/next/client-libraries-cpp/#connection-urls) section of CPP clients.
* Single-source: the [TLS authentication](https://pulsar.apache.org/docs/next/security-tls-authentication#configure-tls-authentication-in-pulsar-clients) topic has introduced the examples of all clients.